docs/reference/projectyml.rst

```diff
@@ -54,6 +54,34 @@ The code snippet below provides an example of a complete ``project.yml`` file:
       index_urls:
         - https://pypi.org/simple/
 
+
+.. _specify_location_pip:
+
+Specify locations from where V2 modules will be installed
```

Comment:
This section needs a lot more details and structure if we want to make sure that it's clear to the end user. Or we can also drop it for now and create a ticket for this, because it's technically outside of the scope of this ticket.

* We should split this in sub-sections for each of the different alternatives with a section above these three sub-section that gives an short overview of the three different approaches.
* We should include snippets when we refer to setting a specific config option.

Reply:
I will add it now.